docs/source/contributing/developing.adoc

```diff
@@ -221,12 +221,17 @@ $ make integration
 
 NOTE: By default `make integration` only runs tests which are tagged as `@basic`.
 
-To run all the test, use the following command:
+To run all tests, use the following command:
 
 ----
 $ make integration_all
 ----
 
+[NOTE]
```

Comment:
Thanks. @robin-owen will have to rebase anyways, but at least you will not forget again :-)